src/main/scala/com/ubirch/kafka/metrics/PrometheusMetricsHelper.scala

```diff
@@ -13,12 +16,12 @@ object PrometheusMetricsHelper extends LazyLogging {
   final private val maxAttempts = 3
 
   def create(port: Int): HTTPServer = {
-    DefaultExports.initialize()
+    val customCollectionRegistry = createCustomCollectorRegistry
 
     @tailrec
     def go(attempts: Int, port: Int): HTTPServer = {
       try {
-        new HTTPServer(port)
+        new HTTPServer(new InetSocketAddress(port), customCollectionRegistry)
```

Comment:
What would be interesting here is to offer the capacity to have both. The user of the helper should be able to start the collector that he/she prefers better.